test/metabase/db/liquibase_test.clj

```diff
@@ -128,3 +132,24 @@
             (deliver released true)
             (testing "The lock was released before the migration finished"
               (is (not @locked?)))))))))
+
+(deftest auto-release-session-lock-test
```

Comment:
@crisptrutski this does it. If you run this test without the dep it will fail.

Reply:
Looks great!